docs/_posts/murat-gunay/2022-02-18-distilbert_base_sequence_classifier_airlines_en.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: model
+title: Sentiment Analysis of Airline Comments
+author: John Snow Labs
+name: distilbert_base_sequence_classifier_airlines
+date: 2022-02-18
+tags: [airlines, distilbert, sequence_classification, en, open_source]
+task: Text Classification
+language: en
+edition: Spark NLP 3.4.0
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model was imported from `Hugging Face ([link](https://huggingface.co/tasosk/distilbert-base-uncased-airlines)) and it's been trained on tasosk/airlines dataset, leveraging `Distil-BERT` embeddings and `DistilBertForSequenceClassification` for text classification purposes. The model classifies `YES` or `NO` sentiments of texts related to airlines reviews.
```

Comment:
Unclosed quotes

Reply:
The model classifies texts into two categories: `YES` for positive comments, and `NO` for negative.